src/intentHandlers/ProjectStatus.js

```diff
@@ -282,3 +386,18 @@ const returnIssues = (JiraResponse) => {
     }
     return result;
 };
+
+// return a list of new-line ended JIRA ticket information in the form 'JIRA_KEY - JIRA_ASSIGNEE - JIRA_SUMMARY (TIME_AGO)'
+const returnIssuesWithAssignee = (JiraResponse) => {
```

Comment:
Starting function names with `return` is a bit unusual. Normally, we prefer that non-mutating / side-effect free methods are noun names (e.g. `issuesWithAssignee`) rather than verbs (`return...`).

Reply:
Done.